polygon-digital-carbon/matchstick.yaml

```diff
@@ -0,0 +1,3 @@
+testsFolder: tests
```

Comment:
If you enable these lines in the GitHub Actions workflows then it will actually run the tests as a check on every PR: https://github.com/KlimaDAO/klima-subgraph/blob/main/.github/workflows/ci.yaml#L50
https://github.com/KlimaDAO/klima-subgraph/blob/main/.github/workflows/deploy.yaml#L54

Reply:
nice, looks like it worked